vue/dashboard/public/index.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width,initial-scale=1.0">
+    <link rel="icon" href="<%= BASE_URL %>favicon.ico">
+    <link href="https://unpkg.com/leaflet@1.3.1/dist/leaflet.css" rel="stylesheet">
```

Comment:
This is not required, right?

Reply:
It its, but only because I didn't take the time to check how to load the styles with the webpack they provide.

It's kind of redundant, since Leaflet is specified as an npm dependency.

Reply:
Turns out you can just `import 'leaflet/dist/leaflet.css';`